test/rhsmlib_test/base.py

```diff
@@ -108,7 +108,7 @@ def setUp(self):
         self.addCleanup(self.stop_server)
         sender_patcher = mock.patch("rhsmlib.client_info.DBusSender.get_cmd_line")
         self.mock_sender_get_cmd_line = sender_patcher.start()
-        self.mock_sender_get_cmd_line.return_value = "nose-unit-test"
+        self.mock_sender_get_cmd_line.return_value = "pytest-unit-test"
```

Comment:
maybe we can simply drop the name of the test runner and make it simply "unit-test", as neither nose nor pytest matter for this